src/background.js

```diff
@@ -322,7 +322,58 @@ function stripFbclid(url) {
   return strippedUrl.href;
 }
 
+async function tabUpdateListener (tabId, changeInfo, tab) {
+  await updateBrowserActionIcon(tab.url);
+}
+
+async function areAllStringsTranslated () {
+  const browserUILanguage = browser.i18n.getUILanguage();
+  if (browserUILanguage && browserUILanguage.startsWith("en")) {
+    return true;
+  }
+  const enMessagesPath = browser.extension.getURL("_locales/en/messages.json");
+  const resp = await fetch(enMessagesPath);
+  const enMessages = await resp.json();
+
+  for (const key of Object.keys(enMessages)){
+    // TODO: this doesn't check if the add-on messages are translated into
+    // any other browser.i18n.getAcceptedLanguages() options ... but then,
+    // I don't think browser.i18n let's us get messages in anything but the
+    // primary language anyway? Does browser.i18n.getMessage automatically
+    // check for secondary languages?
+    const enMessage = enMessages[key].message;
+    const translatedMessage = browser.i18n.getMessage(key);
+    if (translatedMessage == enMessage) {
+      return false;
+    }
+  }
+  return true;
+}
+
+async function updateBrowserActionIcon (url) {
+  const fullyTranslated = await areAllStringsTranslated();
+  if (!fullyTranslated) {
+    browser.browserAction.disable();
+    return;
+  }
+  if (isFacebookURL(url)) {
```

Comment:
When opening a new Facebook Container tab using the Multi-Account Containers popup, the `url` will by default be set to `about:newtab` and thus the popup will get set to `panel2.html` and show the "not in Facebook Container" text. This would probably need to additionally check the `cookieStoreId` to prevent that.

Reply:
Thanks @stoically. Can you file a follow-up issue for that? (Or is there an existing issue that we could fix to prevent the bug?) I'd like to merge this to release 1.6 for the majority of users who are using only one or the other of the add-ons.

Reply:
Sure, opened #215. Though, the new tab longPressBehavior is available with just FC installed.